ansible/configs/ocp4-disconnected-osp-lab/default_vars.yml

```diff
@@ -231,34 +232,36 @@ query_subnet_cidr: "[?name=='{{ ocp_network }}'].subnet_cidr"
 # You can create as many as you want, but at least one is required.
 # Use the name of the networks where appropriate in the instance list
 networks:
-  - name: ocp
-    shared: "false"
-    subnet_cidr: 192.168.47.0/24
-    gateway_ip: 192.168.47.1
-    allocation_start: 192.168.47.10
-    allocation_end: 192.168.47.254
-    dns_nameservers: []
-    create_router: true
+- name: ocp
+  shared: "false"
+  subnet_cidr: 192.168.47.0/24
+  gateway_ip: 192.168.47.1
+  allocation_start: 192.168.47.10
+  allocation_end: 192.168.47.254
+  dns_nameservers:
```

Comment:
I know why you added this based on the chat earlier, but I don't understand why we need it. We should just let the OpenStack DNS servers do this resolution for us. It is how we have been doing it and it has worked fine - not sure what broke. 

I seem to remember not doing this because it broke name resolution inside the tenant. Like you were no longer able to resolve `example.com` because Google DNS has NFC what that is.